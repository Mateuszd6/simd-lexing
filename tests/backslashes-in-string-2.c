struct foobar
{
    int a;
    int b;
    itn c;
    int d;
};

/* Move to the end */
int
main(int argc, char** argv)
{
    char const* foo = "And this is a very long string\
which has backslashes and newlines to make sure it calculates\
line numbers and stuff. Hello cruel world!";
