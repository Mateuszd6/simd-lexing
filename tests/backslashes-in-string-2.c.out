Parsed: 16 lines, 19 ids, 0 strings, 0 chars, 0 ints, 0 hex,   0 floats,    0 //s, 1 /**/s,   0 #foo
